### PR TITLE
restructuring for use of more than one database adapter / config

### PR DESCRIPTION
**Pull request by joelmdesouza**
-created Adapter, type interface adapter

### CODE REVIEW

File: config.go [hunk 1]
```diff
@@ -47,6 +48,9 @@ type Prest struct {
 // PrestConf config variable
 var PrestConf *Prest
 
+// Adapter adapter variable
+var Adapter adapters.Adapter
```
Comment (felipeweb) [inline config.go]: adapter don't need to be global can be a field of Prest struct

**Review by felipeweb — CHANGES_REQUESTED** [review]

**Review by avelino — APPROVED** [review]
LGTM

**Review by crgimenes — APPROVED** [review]
LGTM

**Comment by lerrua** [comment]
@joelmdesouza do you need some help to solve file conflicts? This PR is blocking https://github.com/prest/prest/pull/218 and https://github.com/prest/adapters/pull/15.

**Comment by joelmdesouza** [comment]
@lerrua done

**Comment by joelmdesouza** [comment]
@lerrua 
When it was committed there was no conflict, the conflict occurred after merge another PR.
I only heard of the conflict after you warned me.
Sorry for the delay.
Thank you for the warning.

**Pull request merged**